sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -243,6 +243,7 @@ public string CorrelationId
         ///     <item><description>Stream</description></item>
         ///     <item><description>Uri</description></item>
         ///     <item><description>TimeSpan</description></item>
+        ///     <item><description>byte[]</description></item>
```

Comment:
Should we also add `ArraySegment<byte>`?

Reply:
I left that off intentionally, because when you receive from the service, it gets transformed into `byte[]` in all cases because there is no way to differentiate between the types.   You can add an array segment when sending, but for all intents and purposes it becomes `byte[]`